src/index.js

```diff
@@ -187,7 +187,7 @@ module.exports = () => ({
       <h1 class="text-primary">TestCafe Test Summary</h1>
       <br>
       <div class="client-logo" style="padding:15px"></div>
-      <div class="bg-primary" style="padding:15px">
+      <div class="bg-primary" style="padding:15px;box-shadow: 10px 10px 10px 10px #00000033;border-radius: 10px;"">
```

Comment:
wow, that is a lot of shadow! isn't 6 or 5 px enough? :)

Reply:
Hi @timnederhoff ,

Just I did to looks clean. But it is really awesome. Attached screenshot for your reference. 
<img width="1246" alt="Screenshot 2020-02-11 at 2 24 26 PM" src="https://user-images.githubusercontent.com/41051957/74222410-6f2ca600-4cda-11ea-91b7-1130cfcdfeb0.png">

Please review and approve.

Reply:
please at lease have the color of the shadow similor for summary and results table.
ps. don't forget fix the failing test (see discussion)

Reply:
Hi @timnederhoff,

Modified the shadows to have 6px for all. Now all the styles will look like below. 

<img width="1205" alt="Screenshot 2020-02-17 at 12 45 41 PM" src="https://user-images.githubusercontent.com/41051957/74631619-6b3dcf80-5183-11ea-8cfd-a482ac108621.png">
<img width="1202" alt="Screenshot 2020-02-17 at 12 46 10 PM" src="https://user-images.githubusercontent.com/41051957/74631648-7bee4580-5183-11ea-9ae6-1cd6fac0d929.png">


And also I edited the report-with-color.html file for your reference. 

From your last comment,>> ps. don't forget fix the failing test (see discussion)...

I don't understand what did you mean here. Which failing test did you mean to say? 

Are you meant to say, make all the test cases to have only passed? If so which file should I make changes so that it will reflect?

And also please do let me know how to make a run of test.js file. 

I tried this way but failed with following error
```selvarajan@Autos-MacBook-Pro testcafe-reporter-html % node test/test.js   
/Users/selvarajan/PycharmProjects/Open-Source-Contributions/testcafe-reporter-html/test/test.js:6
it('Should produce report with colors', () => {
^

ReferenceError: it is not defined
    at Object.<anonymous> (/Users/selvarajan/PycharmProjects/Open-Source-Contributions/testcafe-reporter-html/test/test.js:6:1)
    at Module._compile (internal/modules/cjs/loader.js:1151:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1171:10)
    at Module.load (internal/modules/cjs/loader.js:1000:32)
    at Function.Module._load (internal/modules/cjs/loader.js:899:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
selvarajan@Autos-MacBook-Pro testcafe-reporter-html % 
```

Thanks, a bunch for the quick review comments,
Selvarajan

Reply:
the above screenshot shows a border shadow of 6px for the summary and tests table while the code is set to 10px. but it is ok for me. the error you get from running test.js is probably because you forgot to run `npm install` first. Run the test with `npm test` command